cluster/addons/dns/kube2sky/Changelog

```diff
@@ -1,3 +1,9 @@
+## Version 1.15 (Apr 7 2016 Lucas Käldström <lucas.kaldstrom@hotmail.co.uk>)
+ - No code changes since 1.14
```

Comment:
what version was 1.14 built with? probably want to include that this is using go 1.6